r5jxrm/r5jxrm/Osszeg.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using _r5jxrm_;
+using System.Xml.Serialization;
+
+namespace _r5jxrm_
+{
+    public class Osszeg
+    {
+        //Az XML fájlom felépítése
+        //Mindegyik kapott default értéket, hogy nehogy null-t adjon valamelyik helyen (pl amikor létrehozom az objektumot)
+        public double nyeremeny { get; set; } = 0;
```

Comment:
a `double` value type, ami konstruktor hívás nélkül is kap értéket ami 0, szóval redundáns az inicializáció. És mivel value type nem lehet neki null értéke. Null értéke csak a double?-nek lenne, ami nullable double, ami valójában pedig egy Nullable<T> objektumba csomagolja be a dolgot.